### ISSUE TITLE
Initialize in Angular doesnt work since version 17.0.0 -  TypeError: Keycloak is not a function

### ISSUE THREAD

**Issue by stylepatrick** [keycloak/keycloak#11571]
### Describe the bug

Initialize Keycloak connection in Angular doesnt work since Version 17.0.0.
Release version 16.1.1 works.

Following code used:
`        const keycloakAuth = Keycloak({
            url: 'https://server/auth',
            realm: 'realm',
            clientId: 'client'
        });`

Error:

![image](https://user-images.githubusercontent.com/52086629/164446050-bb9af10f-34ff-4deb-929d-29a5ba248bc7.png)

### Version

Keycloak 17.0.0
keycloak-js: 17.0.0
typescript: 4.5.4
angular: 13.1.2

**Comment by stylepatrick** [comment]
Still getting the same issue. Is there any solution or workaround.
The same implementation works with versions < 17.0.0.

**Comment by micksi** [comment]
It looks like the issue might be that the declarations file is exposing a 2 separate things called Keycloak. One is the function you have been using in previous version, which has been deprecated, and instead it uses a class called Keycloak. The function declaration and the deprecated flag should be removed in the declarations file since it only causes problems with name collisions. 

TL;DR use `new Keycloak(config)` in stead of `Keycloak(config)`

**Comment by stylepatrick** [comment]
This is how my changed code looks:
` const keycloakAuth: any = new Keycloak(configuration);`

Tried to change the `Keycloak(config)` to `new Keycloak(config)` but the compiler gives me an error:
`Only a void function can be called with the 'new' keyword.`

![image](https://user-images.githubusercontent.com/52086629/175868439-551ee2ec-05b1-48e0-b812-c10d850b380c.png)

This is how i import Keycloak:
`import * as Keycloak_ from 'keycloak-js';`
`const Keycloak = Keycloak_;`

Is there anything wrong on my code? Like i said the same code works with versions < 17.0.0.

**Comment by jonkoops** [comment]
I am unable to reproduce this issue using the latest version of Keycloak JS (19.0.1). Both the function style construction and `new` keyword work.
